client/views/actions/invite.tpl

```diff
@@ -1,9 +1,7 @@
-<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{from}}</span>
-invited
-{{#if invitedYou}}
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{from}}</span>invited
+{{~#if invitedYou~}}
 	you
```

Comment:
This one will need a space, but it'll be stripped by the "~" at the end of the next line. Maybe stick this in a span, and put the sapces in there?